include/ceed-impl.h

```diff
@@ -131,11 +131,34 @@ struct CeedQFunction_private {
   CeedInt numinputfields, numoutputfields;
   int (*function)(void*, CeedInt, const CeedScalar *const*, CeedScalar *const*);
   const char *focca;
+  bool fortranstatus;
```

Comment:
I'm not commited to this particular variable name. It works but I'm open to a better name.

Reply:
I think it would be better to separate the ctx and fortran_ctx. There is no reason why the backend should have access to the fortran_ctx.

Reply:
The backend only would have to consider the fortran_ctx if the backend choses to bypass the Fortran interface when applying the QFunction. I fixed the implementation in the Fortran interface.

Here is the issue: The user provided Fortran function is wrapped in a C function to match the libCEED spec. But QFunctions only have 4 arguments, ctx, number of quad pts, input, and output. In order to have the C wrapped Fortran function match this signature, we need a way to pass in the true Fortran pointer, and ctx is the best place to do that.